src/algo_tx.h

```diff
@@ -75,15 +75,22 @@ struct state_schema {
   uint64_t num_byteslice;
 };
 
-#define MAX_ACCT 2
-typedef uint8_t accounts_t[MAX_ACCT][32];
+#define MAX_ACCT 4
+#define MAX_FOREIGN_APPS 2 
```

Comment:
Based on: https://developer.algorand.org/docs/get-details/parameter_tables/#smart-contract-constraints
Current limits are for foreign apps and foreign assets (but 8 total for accounts+asas+apps) - should this be increased?